src/scss/grommet-core/_objects.article.scss

```diff
@@ -55,6 +55,10 @@
   }
 }
 
+.button.article__control.button--plain.button--icon {
```

Comment:
Can we move the overflow hidden to the existing button selector? See this line:

https://github.com/DerekAhn/grommet/blob/fix/vertical-bars-appearing-on-Article-arrows/src/scss/grommet-core/_objects.article.scss#L67